.circleci/config.yml

```diff
@@ -90,5 +90,5 @@ workflows:
       - build:
           matrix:
             parameters:
-              postgresql_version: [ "10-alpine", "12-alpine" ]
+              postgresql_image: [ "circleci/postgres:10-alpine-ram", "cimg/postgres:13.11" ]
```

Comment:
Why do we use `postgres:13-alpine` above and here we use the circleci images? Just in case there is a special reason. I guess because we used to do in this way.

Reply:
Yeah, that surprised me also, I don't know why! It was already like that... 

Happy to change `postgres:13-alpine` to a CircleCI image, if it makes sense.

We'll need to add version 12 in the tests, because I became aware that this is the version we still use in SaaS :grimacing: